web/html/ParamsApp/php/NewParams.php

```diff
@@ -101,16 +109,17 @@ function isSameResource($prevRow, $currRow) {
     $DT9 = "|9|";
 }
 
-$query = "
```

Comment:
I actually prefer this code, since you can copy the entire string and execute as a sql statement by only replacing the $DataParameterTable, and it's a single assignment.

Reply:
fixed in last commit